server/server.js

```diff
@@ -1,9 +1,9 @@
 require('dotenv').config();
 const express = require('express');
-const configRoutes = require('./routes');
 
 const app = express();
 app.use(express.json());
+const configRoutes = require('./routes');
```

Comment:
why have you changed the order of this import? i am not getting any error for this...